src/sections/support-page/support-channels/support-channels.module.scss

```diff
@@ -0,0 +1,68 @@
+@import '~@bit/bit.base-ui.layout.breakpoints';
+
+.wrapper {
+    margin-bottom: 30px;
+}
+
+
+.mainGrid {
+	gap: 20px;
+    align-items: center;
+    margin-bottom: 150px;
+}
+
+.image {
+    max-width: 100%;
+    margin: 0 auto;
+    display: none;
+    @media screen and (min-width: $br-md) {
+        display: unset;
+    }
+}
+
+.content {
+    margin: 0 auto;
+}
+
+.mainParagraph {
+    margin-bottom: 30px;
+    color: var(--text-muted, #6c707c);
```

Comment:
you can use className `mutedText` from the MutedText component.
(less direct dependencies on global hardcoded css vars)

Reply:
done